test/es6/rlexe.xml

```diff
@@ -322,6 +322,18 @@
   </test>
   <test>
     <default>
+      <files>ES6Super.js</files>
+      <compile-flags>-ES6Classes -ES6Spread -ES6Generators -Off:Deferparse -args summary -endargs</compile-flags>
```

Comment:
Removed in latest rev

---
In reply to: [75547623](https://github.com/Microsoft/ChakraCore/pull/1466#discussion_r75547623) [](ancestors = 75547623,75533634)